src/components/Event.jsx

```diff
@@ -68,6 +68,17 @@ const DummyDiv = styled.div({
   width: "100%"
 });
 
+const PlayerCloseButton = styled.button(props => ({
+  position: "absolute",
+  top: "5px",
+  right: "5px",
+  opacity: "0.5",
+  "border-radius": "50%",
+  zIndex: 10,
+  visibility: props.isFixed ? "visible" : "hidden",
+  "&:hover": { opacity: 0.9 }
+}));
+
```

Comment:
I would change PlayerCloseButton to use semantic-ui-react Button component with a 'close' icon.
It would look something like this:
```suggestion
const PlayerCloseButton = styled(Button)(props => ({
  position: "absolute",
  top: "0",
  right: "0",
  zIndex: 10,
  visibility: props.isFixed ? "visible" : "hidden",
  backgroundColor: "transparent !important",
  padding: "5px !important"
}));
```

![image](https://user-images.githubusercontent.com/37560480/75314453-a475df80-5814-11ea-99a8-cb5822d99c74.png)
I tried to make it look a little bit like youtube's mini-player close button.